expyfun/stimuli/_tracker.py

```diff
@@ -130,7 +130,11 @@ def __init__(self, callback, up, down, step_size_up, step_size_down,
                  change_rule='reversals', x_min=None, x_max=None,
                  repeat_limit='reversals'):
         self._callback = _check_callback(callback)
+        if type(up) != int:
```

Comment:
the nicer check would be `if not isinstance(up, int)`. But going one level further, based on a (long) SciPy discussion the consensus is this is the best int-like check:
```
up = operator.index(up)
```
It automatically converts to `int` things like `np.array(12, np.int32)` and throws a reasonable error for non-int-like

Reply:
Hmm, so tradeoff is between using ``isinstance`` and having a more specific error message or ``operator.index`` and getting a conversion for int-likes? I think I prefer the former since the tracker code is so lengthy to sort through.

Reply:
> I think I prefer the former since the tracker code is so lengthy to sort through.

`up = operator.index(up)` is one line vs the two for `isinstance` check, and works in more cases, but if you really like `isinstance` we can keep it here